cmd/fossa/main.go

```diff
@@ -328,21 +246,27 @@ func defaultCmd(c *cli.Context) {
 			mainLogger.Fatalf("Could marshal analysis results: %s", err.Error())
 		}
 		fmt.Println(string(buildData))
+		return
 	}
 
-	if config.analyzeConfig.noUpload {
-		return
+	s.Stop()
+	s.Suffix = fmt.Sprintf(" Writing configuration...")
+	s.Restart()
+
+	if err := config.WriteConfigFile(&conf); err != nil {
```

Comment:
Does this work if you run the default command but already have a configuration file?

Reply:
Yes, it should normalize the config file